### ISSUE TITLE
dataset

### ISSUE THREAD

**Issue by suke27** [roimehrez/PIRM2018#2]
Hello, do you have training dataset and test dataset for x8 case?

**Comment by roimehrez** [comment]
We have the self validation free for any use with the GT. The test set will
be released after ECCV

בתאריך יום ב׳, 3 בספט׳ 2018, 12:54, מאת suke27 ‏<notifications@github.com>:
